src/main/scala/com/exasol/spark/DefaultSource.scala

```diff
@@ -112,10 +113,16 @@ class DefaultSource
       case SaveMode.ErrorIfExists =>
         if (isTableExist) {
           throw new UnsupportedOperationException(
-            s"""|Table $tableName already exists. And DataFrame write mode is set to
-                |`errorifexists` or `default`. Please use one of other SaveMode
-                |modes: 'append', 'overwrite' or 'ignore'.
-            """.stripMargin
+            ExaError
+              .messageBuilder("E-SEC-3")
+              .message(
+                "Table {{TABLE}} already exists in 'errorifexists' or 'default' write modes."
+              )
+              .parameter("TABLE", tableName)
```

Comment:
```suggestion
                "Table {{TABLE}} already exists in 'errorifexists' or 'default' write modes.", tableName)
```

Reply:
You can use a more compact way to add parameters, but the current is also fine if you like it more

Reply:
Done. It is definitely better for single parameters.